### ISSUE TITLE
the file is too large to upload

### ISSUE THREAD

**Issue by zenppl** [actions-on-google/smart-home-dashboard#6]
ERROR: (gcloud.app.deploy) Cannot upload file [/Users/zenppl/Documents/gitlab/smart-home-dashboard/target/appengine-staging/dashboard-0.1.0-SNAPSHOT.war], which has size [42258238] (greater than maximum allowed size of [33554432]). Please delete the file or add to the skip_files entry in your application .yaml file and try again.

By the way, i use the .yaml you provided in [issue 1](https://github.com/actions-on-google/smart-home-dashboard/issues/4#issuecomment-411194099) ,because there is no .yaml file after `mvn compile` and `mvn appengine:deploy`.

**Comment by Fleker** [comment]
I just pulled the sample again today and while it does seem like the `dashboard-0.1.0-SNAPSHOT.war` is pretty large, it doesn't seem like it's blocking me from deploying the project to App Engine.

**Comment by wxhuang** [comment]
I have the same problem. Is there any way to reduce the package size or increase the maximum upload size?

**Comment by wxhuang** [comment]
Build the frontend first and the problem may be resolved. https://github.com/actions-on-google/smart-home-dashboard/issues/4#issuecomment-419343347

**Comment by zenppl** [comment]
I pulled and deploy again, there is no error this time.

**Comment by ldgmaia** [comment]
I also have this issue. I already pulled the sample 20 times and I got this error every time.
Why is so hard to follow a simple step-by-step?

[INFO] --- appengine-maven-plugin:1.3.2:deploy (default-cli) @ dashboard ---
[INFO] Staging the application to: C:\Users\XX\Desktop\smart-home-dashboard\target\appengine-staging
[INFO] Detected App Engine flexible environment application.
Mar 06, 2019 11:04:12 AM com.google.cloud.tools.appengine.cloudsdk.CloudSdk logCommand
INFO: submitting command: C:\Program Files (x86)\Google\Cloud SDK\google-cloud-sdk\bin\gcloud.cmd app deploy
[INFO] GCLOUD: Services to deploy:
[INFO] GCLOUD:
[INFO] GCLOUD: descriptor:      [C:\Users\XX\Desktop\smart-home-dashboard\target\appengine-staging\app.yaml]
[INFO] GCLOUD: source:          [C:\Users\XX\Desktop\smart-home-dashboard\target\appengine-staging]
[INFO] GCLOUD: target project:  [mydashboarddrain]
[INFO] GCLOUD: target service:  [default]
[INFO] GCLOUD: target version:  [20190306t110414]
[INFO] GCLOUD: target url:      [https://mydashboarddrain.appspot.com]
[INFO] GCLOUD:
[INFO] GCLOUD:
[INFO] GCLOUD: Beginning deployment of service [default]...
[INFO] GCLOUD: ERROR: (gcloud.app.deploy) Cannot upload file [C:\Users\XX\Desktop\smart-home-dashboard\target\appengine-staging\dashboard-0.1.0-SNAPSHOT.war], which has size [42397845] (greater than maximum allowed size of [33554432]). Please delete the file or add to the skip_files entry in your application .yaml file and try again.
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.870 s
[INFO] Finished at: 2019-03-06T11:04:15-05:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.google.cloud.tools:appengine-maven-plugin:1.3.2:deploy (default-cli) on project dashboard: Execution default-cli of goal com.google.cloud.tools:appengine-maven-plugin:1.3.2:deploy failed: Non zero exit: 1 -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException

**Comment by Fleker** [comment]
Is everyone in this thread using Windows for development?

**Comment by ldgmaia** [comment]
> Is everyone in this thread using Windows for development?

I am.

**Comment by Fleker** [comment]
I'm going to close this issue until there's more information, as right now I can't reproduce an issue.

**Comment by Jeremy-Busuttil** [comment]
How did you manage to fix this issue? I am new to GCP and Im trying to upload an API to google app engine however a file Im using is too large and its not letting me put it up

**Comment by esimmons92** [comment]
I am also facing this issue

which has size [70450035] (greater than maximum allowed size of [33554432]

VIA mac
